### DISCUSSION TITLE
job chips not displaying salary range?

### DISCUSSION THREAD

**Discussion by ICmike** [bullhorn/career-portal#402] [Q&A]
Hi

I've just managed (with some help) to add salary to job chips which works great for fixed salary jobs 

But if its a salary range like 25,000 to 35,000 or is up to 35,000, it also doesn't display?

Is there something i'm missing or need to add to achieve this?

Thanks
Mike

**Comment by charlesabarnes** [comment]
This isn't an intended feature at this time.

**Reply by ICmike** [reply]
Is there any work around that you know of to get this done?

**Reply by charlesabarnes** [reply]
Hello, how is this data being stored for the range?
